### ISSUE TITLE
Crash with "Offset is unspecified" exception

### ISSUE THREAD

**Issue by Occhioverde** [MM2-0/Kvaesitso#123]
Good morning.

As per the title, I would like to report that on my Huawei tablet Kvæsitso sometimes crashes reporting the "Offset is unspecified" exception.

To give a bit of context in case it can be useful, the crash seems to happen when I slide down the home screen two times in a row, noticing that the app drawer doesn't appear after a second or two.

Exception traceback:

```
java.lang.IllegalStateException: Offset is unspecified
	at androidx.compose.ui.geometry.Offset.getX-impl(Offset.kt:67)
	at androidx.compose.ui.geometry.Offset.minus-MK-Hz9U(Offset.kt:168)
	at androidx.compose.ui.input.pointer.PointerEventKt.positionChangeInternal(PointerEvent.kt:816)
	at androidx.compose.ui.input.pointer.PointerEventKt.positionChangeIgnoreConsumed(PointerEvent.kt:810)
	at androidx.compose.foundation.gestures.DragGestureDetectorKt.verticalDrag-jO51t88(DragGestureDetector.kt:374)
	at androidx.compose.foundation.gestures.DragGestureDetectorKt$verticalDrag$1.invokeSuspend(Unknown Source:15)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTaskKt.resume(DispatchedTask.kt:178)
	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:166)
	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:397)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl(CancellableContinuationImpl.kt:431)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$default(CancellableContinuationImpl.kt:420)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:328)
	at androidx.compose.ui.input.pointer.SuspendingPointerInputFilter$PointerEventHandlerCoroutine.offerPointerEvent(SuspendingPointerInputFilter.kt:563)
	at androidx.compose.ui.input.pointer.SuspendingPointerInputFilter.dispatchPointerEvent(SuspendingPointerInputFilter.kt:455)
	at androidx.compose.ui.input.pointer.SuspendingPointerInputFilter.onPointerEvent-H0pRuoY(SuspendingPointerInputFilter.kt:468)
	at androidx.compose.ui.input.pointer.Node.dispatchMainEventPass(HitPathTracker.kt:310)
	at androidx.compose.ui.input.pointer.NodeParent.dispatchMainEventPass(HitPathTracker.kt:179)
	at androidx.compose.ui.input.pointer.HitPathTracker.dispatchChanges(HitPathTracker.kt:98)
	at androidx.compose.ui.input.pointer.PointerInputEventProcessor.process-BIzXfog(PointerInputEventProcessor.kt:97)
	at androidx.compose.ui.platform.AndroidComposeView.sendMotionEvent-8iAsVTc(AndroidComposeView.android.kt:1280)
	at androidx.compose.ui.platform.AndroidComposeView.handleMotionEvent-8iAsVTc(AndroidComposeView.android.kt:1230)
	at androidx.compose.ui.platform.AndroidComposeView.dispatchTouchEvent(AndroidComposeView.android.kt:1169)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2974)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2661)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2974)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2661)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2974)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2661)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2974)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2661)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2974)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2661)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2974)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2661)
	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:549)
	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1953)
	at android.app.Activity.dispatchTouchEvent(Activity.java:3559)
	at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:502)
	at android.view.View.dispatchPointerEvent(View.java:12027)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:5304)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5093)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4588)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4641)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4607)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4734)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4615)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4791)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4588)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4641)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4607)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4615)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4588)
	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:7195)
	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:7169)
	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:7130)
	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:7351)
	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:379)
	at android.os.Looper.loop(Looper.java:144)
	at android.app.ActivityThread.main(ActivityThread.java:7529)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:245)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:921)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [androidx.compose.ui.platform.MotionDurationScaleImpl@22bf65, androidx.compose.runtime.BroadcastFrameClock@23a743a, StandaloneCoroutine{Cancelling}@12958eb, AndroidUiDispatcher@874a648]
```

Device info printed by Kvæsitso:

```
APP.VERSION : 1.13.1-20220712
APP.VERSIONCODE : 2022071200
APP.SIGNATURE : rx1fSnL7r5/OMoFC0e1KPqTndXQ=
LAUNCHER.APP : com.huawei.android.launcher
TIMEZONE : Europe/Rome
VERSION.RELEASE : 8.0.0
VERSION.INCREMENTAL : 298(OCEC432)
VERSION.SDK.NUMBER : 26
BOARD : BAH2
BOOTLOADER : unknown
BRAND : HUAWEI
CPU_ABI : arm64-v8a
CPU_ABI2 : 
DISPLAY : BAH2-W19 8.0.0.298(OCEC432)
FINGERPRINT : HUAWEI/BAH2-W19/HWBAH2:8.0.0/HUAWEIBAH2-W19/298(OCEC432):user/release-keys
HARDWARE : hi6250
HOST : cn-central-1b-3d4366e431593657897564-768571480-jllh1
ID : HUAWEIBAH2-W19
MANUFACTURER : HUAWEI
MODEL : BAH2-W19
PRODUCT : BAH2-W19
TAGS : release-keys
TIME : 1593663394000
TYPE : user
```

**Comment by bartlibert** [comment]
I also have this issue, but I cannot reproduce it consistently.


## Description

*Please provide as many information about the crash as possible (What did you do before the crash happened? Steps to reproduce?)*

## Strack trace

```
java.lang.IllegalStateException: Offset is unspecified
	at androidx.compose.ui.geometry.Offset.getX-impl(Offset.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.fromParentPosition-MK-Hz9U(LayoutNodeWrapper.kt:2)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
	at androidx.compose.ui.node.LayoutNodeWrapper.ancestorToLocal-R5De75A(LayoutNodeWrapper.kt:3)
[33 lines truncated]
```

## Device info
APP.VERSION : 1.14.5-20220824<br>APP.VERSIONCODE : 2022082402<br>APP.SIGNATURE : rx1fSnL7r5/OMoFC0e1KPqTndXQ=<br>LAUNCHER.APP : de.mm20.launcher2.release<br>TIMEZONE : Europe/Brussels<br>VERSION.RELEASE : 12<br>VERSION.INCREMENTAL : G781BXXU4FVG6<br>VERSION.SDK.NUMBER : 31<br>BOARD : kona<br>BOOTLOADER : G781BXXU4FVC4<br>BRAND : samsung<br>CPU_ABI : arm64-v8a<br>CPU_ABI2 : <br>DISPLAY : SP1A.210812.016.G781BXXU4FVG6<br>FINGERPRINT : samsung/r8qxeea/r8q:12/SP1A.210812.016/G781BXXU4FVG6:user/release-keys<br>HARDWARE : qcom<br>HOST : 21DKGB14<br>ID : SP1A.210812.016<br>MANUFACTURER : samsung<br>MODEL : SM-G781B<br>PRODUCT : r8qxeea<br>TAGS : release-keys<br>TIME : 1659090195000<br>TYPE : user

**Comment by lgwapnitsky** [comment]
Getting this on Pixel 6 Pro with Android 13

**Comment by MM2-0** [comment]
I think this will be fixed in the next update (1.21), please reopen if it happens again
